x-pack/plugins/security/index.js

```diff
@@ -112,11 +107,17 @@ export const security = (kibana) => new kibana.Plugin({
     // automatically assigned to all routes that don't contain an auth config.
     server.auth.strategy('session', 'login', 'required');
 
-    const auditLogger = new SecurityAuditLogger(server.config(), new AuditLogger(server, 'security'));
-
     // exposes server.plugins.security.authorization
     initAuthorizationService(server);
 
+    watchStatusAndLicenseToInitialize(xpackMainPlugin, plugin, async (license) => {
```

Comment:
I moved this because the `registerPrivilegesWithCluster` function is now dependent on the `server.plugins.authorization` that `initAuthorizationService` exposes.